app/services/stats/r901_offender_sar_cases_report.rb

```diff
@@ -28,27 +28,27 @@ def self.description
 
     def initialize(**options)
       super(**options)
-      @case_scope = options[:case_scope] || Case::SAR::Offender.all
-    end 
+      @case_scope = options[:case_scope] || Case::Base.offender_sar.all
```

Comment:
Suggest to use the `where(type: ["Case::SAR::Offender", "Case::SAR::OffenderComplaint"])`  instead of changing the scope of offender_sar flag in case base model